src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Colors[] values = Colors.values();
+        return values[new Random().nextInt(values.length)];
```

Comment:
```suggestion
    private static final Random random = new Random();
    
    public Colors getRandomColor() {
        Colors[] values = Colors.values();
        return values[random.nextInt(values.length)];
```
lets make random global
this way the random object will be created only once, not every time you invoke this method